jep/221/README.adoc

```diff
@@ -65,15 +65,25 @@ endif::[]
 == Abstract
 
 This JEP introduces a new system which enables Jenkins plugin maintainers to add Continuous Delivery (CD) to their projects.
-It is:
-
-* Opt-in: plugin maintainers can choose to use this
-* Simple to configure: very little needs to be changed in order to enable CD
-* Trusted CI/CD: deployments are produced from a single source of truth, which is managed by 
-the Jenkins project
 
 == Specification
 
+=== Features
+
+* *Trusted CI/CD*:
```

Comment:
beware that this might be construed more specifically as a particular server

Reply:
@jglick  
Can you suggest change?